oam/install_scripts/columnstore-post-install.in

```diff
@@ -151,17 +173,28 @@ rm -f $lockdir/columnstore
 #check and get amazon env variables
 aws=`which aws 2>/dev/null`
 if [ -z "aws" ]; then
-	MCSgetCredentials.sh >/dev/null 2>&1
+    MCSgetCredentials.sh >/dev/null 2>&1
 fi
 
 postConfigure
 systemctl start mariadb-columnstore
 
-while  [ -z  $(pgrep -x controllernode) ];
+while  [ -z  "$(pgrep -x controllernode)" ];
 do
-    sleep 2
+    sleep 1
 done
  
 dbbuilder 7 > $tmpDir/dbbuilder.log
-exit 0
 
+if [ $stop_mysqld -eq 1 ];then
+    # Make sure we stop mariadb since it wasn't running prior to columnstore installation
+    systemctl cat mariadb.service > /dev/null 2>&1
+    if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
+        systemctl stop mariadb.service > /dev/null 2>&1
+    else
+        pkill mysqld
+    fi
+
+fi
```

Comment:
Let us not forget to stop mariadb if we started it.

Reply:
Need to test this on non-systemd mariadb. Don't merge unless you're sure this works. we are starting mysqld_safe so i need to make sure this doesn't need to change.

Reply:
Confirmed system works as expected when mariadb system unit does not exist